vigilance/cli.py

```diff
@@ -29,11 +30,11 @@ def main(configFile): #pylint: disable=missing-docstring, invalid-name
     except yaml.YAMLError:
         raise ConfigurationParsingError('Could not load configuration file as yaml')
     except Invalid as ex:
-        raise ConfigurationParsingError('Invalid configuration schema: {}'.format(ex.message))
-    unknownSuites = [suite for suite in suites.iterkeys() if suite not in QualitySuite.available_suites()]
+        raise ConfigurationParsingError('Invalid configuration schema: {}'.format(ex.msg))
```

Comment:
what sparked this name change

Reply:
Was getting "'Invalid' object has no attribute 'message'" errors in py3. But both py2 and py3 have the 'msg' attribute.